packages/docs/src/components/documentation.tsx

```diff
@@ -166,6 +169,19 @@ export class Documentation extends React.PureComponent<IDocumentationProps, IDoc
         Utils.safeInvoke(this.props.onComponentUpdate, activePageId);
     }
 
+    private updateTitle(){
+        // Update the page <title> when the hash changes
+        let title:string;
+        if(this.state.activePageId === 'blueprint' && this.state.activeSectionId === 'blueprint'){
+            const indexTitle: string = 'Blueprint - Documentation';
+            title = indexTitle;
+        } else {
+            const newTitle: string = document.getElementsByClassName('docs-title')[0].textContent;
```

Comment:
Again, no need to explicitly specify the type.

Double quotes

And I'd prefer a null check before accessing indices in this: `document.getElementsByClassName('docs-title')`